framework/packages/abstract-std/src/objects/account/account_trace.rs

```diff
@@ -22,6 +22,7 @@ impl KeyDeserialize for &AccountTrace {
 
     #[inline(always)]
     fn from_vec(value: Vec<u8>) -> StdResult<Self::Output> {
+        let value = value.into_iter().filter(|b| *b > 32).collect();
```

Comment:
This is good and all, but it feels like a hack that works with current functions. Can't we just serialize the AccountTrace to our format and then use a simple deserialization function.
We should also provide a better serialization function because the one we have currently is very weird for our use-case

Reply:
> This is good, but it feels like a hack that works with current storage. We should also provide a better serialization function because the one we have currently is very weird for our use-case. For example, simply use the Display impl to ser-deser ?

Yes, as we discussed we shouldn't use this type in state and replace it with string

Reply:
Another solution, for composite keys to work better, we could use a maximum trace length for accounts